### DISCUSSION TITLE
Vercel pricing button style issue on hover

### DISCUSSION THREAD

**Discussion by ebraj** [vercel/community#3583] [Bug Report]
### Page to Investigate

https://vercel.com/pricing

### Steps to Reproduce

Steps :
1. Visit the pricing page of vercel - https://vercel.com/pricing
2. On the Hobby pricing card, hover over the `Start Deploying` button, you will see the see the style issue 

Screenshot :
<img width="507" alt="Screenshot 2023-08-12 at 11 13 57 PM" src="https://github.com/vercel/community/assets/61580196/b6cb1a8c-3b90-446e-87bc-036c94fb1acc">

**Comment by leerob** [comment]
Thank you, we will fix this right now!
